packtools/catalogs/htmlgenerator/v2.0/config-labels.xml

```diff
@@ -355,4 +361,16 @@
         <name lang="pt">Acessado</name>
         <name lang="es">Accedido</name>
     </term>
+    <term>
+        <name>Date of issue</name>
+        <name lang="en">Date of issue</name>
+        <name lang="pt">Data do Fascículo</name>
+        <name lang="es">Fecha del número</name>
+    </term>
+    <term>
+        <name>Publication in SciELO</name>
+        <name lang="en">Publication in SciELO</name>
+        <name lang="pt">Publicação em SciELO</name>
+        <name lang="es">Publicación en SciELO</name>
+    </term>
```

Comment:
Não usar Publication in SciELO senão vai ficar assim para todos os sites até para aqueles que usam a metodologia mas não podem ser chamados de SciELO. Usar no lugar publicação online.